tests/reducers/activeOrderReducer.test.js

```diff
@@ -146,21 +146,21 @@ describe('Active Order Reducer', () => {
     expect(newState.activeOrders[0].status).toEqual(action.action);
   });
 
-  it('should set the DISCARD_EDIT order action', () => {
-    const initialState = {
-      activeOrders: [{
-        drugName: 'panadol',
-        status:'DISCONTINUE'
-      }]
-    };
-
-    const action = {
-      type: SET_ORDER_ACTION,
-      action: 'DISCARD_EDIT',
-    };
-    const newState = activeOrderReducer(initialState, action);
-    expect(newState.activeOrders[0].status).toEqual(initialState.activeOrders[0].status);
-  });
+  // it('should set the DISCARD_EDIT order action', () => {
```

Comment:
why is this test commented out?

Reply:
We had an issue with a recently merged pull request which resulted in failing tests because of this code section. Another pull request has been raised to fix this, that is why I commented out the test code section.

Reply:
@fuchodeveloper  it's usually good to either refactor the test to make it pass, or remove it entirely if it is redundant, but never commit commented out code.

Reply:
@kwahalf 
Comments removed as the fix is in a different pull request